hazelcast/src/main/java/com/hazelcast/map/impl/operation/EvictBatchBackupOperation.java

```diff
@@ -68,6 +70,17 @@ public void run() {
         }
     }
 
+    @Override
+    public ExceptionAction onInvocationException(Throwable throwable) {
+        if (throwable instanceof WrongTargetException) {
```

Comment:
Looks good. Could we just have a short comment here?

Reply:
done